Feature/010 create order entity

wangyiqunumich opened the pull request:
Added setters and getters.

wangyiqunumich committed:
Test commit from Yiqun

wangyiqunumich committed:
Test commit from Yiqun

wangyiqunumich committed:
Merge branch 'main' of https://github.com/JerryZhao1025/TradeMarket

wangyiqunumich committed:
Created the fields of Order under Entity. Modified Post as well to pass
compiler check.
Yiqun(Huaiting) 20220824

wangyiqunumich committed:
Added getters and setters

wangyiqunumich committed:
Modified the type of the time stamp for creation.

JerryZhao1025 commented on src/main/java/com/laioffer/trademarket/entity/Order.java:
```diff
@@ -1,4 +1,84 @@
+
+    private boolean paid;
+
+    private boolean shipped;
```
把这两个paid和shipped给去了，之后再做。

JerryZhao1025 commented on src/main/java/com/laioffer/trademarket/entity/Order.java:
```diff
@@ -1,4 +1,84 @@
+
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private java.sql.Timestamp timestampAtCreation;
```
直接用Timestamp type 应该就可以了

JerryZhao1025 commented on src/main/java/com/laioffer/trademarket/entity/Order.java:
```diff
@@ -1,4 +1,84 @@
+
+    @ManyToOne
+    @JsonIgnore
+    private User user;
```
把这一个user改成buyer和saller，type均为User

wangyiqunumich committed:
Updated the SQL table name; replace user with buyer and seller; update
type of time field; modified the getters and setters accordingly.
Updated Post to be empty as talked with Jinyang.

JerryZhao1025 commented on src/main/java/com/laioffer/trademarket/entity/Order.java:
```diff
@@ -1,4 +1,74 @@
+    private Post post;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Timestamp timestampAtCreation;
```
考虑改个名字啥的，比如createdTime

JerryZhao1025 reviewed (commented):
基本ok了

wangyiqunumich committed:
Modified the name of time variable from timestampAtCreation to
createdTime, as talk with Jinyang.

JerryZhao1025 reviewed (dismissed):
+1

wangyiqunumich committed:
Merge branch 'main' into feature/010-create_order_entity

JerryZhao1025 reviewed (approved):
+1
